device.x

```diff
@@ -45,6 +45,7 @@ PROVIDE(USART2 = DefaultHandler);
 PROVIDE(EXTI_LINE15_10 = DefaultHandler);
 PROVIDE(RTC_ALARM = DefaultHandler);
 PROVIDE(USBFS_WKUP = DefaultHandler);
+PROVIDE(EXMC = DefaultHandler);
```

Comment:
Note that this file is auto-generated, so in order to preserve this change across updates, it needs to be added to SVD (for example, via a patch: https://github.com/riscv-rust/gd32vf103-pac/blob/master/patches/gd32vf103.yaml)